### ISSUE TITLE
Install from .zip tile doesn't work from home screen widget

### ISSUE THREAD

**Issue by MartijnKaijser** [phil65/skin.estuary#9]

**Comment by phil65** [comment]
core issue.

**Comment by Razzeee** [comment]
We probably have no chance to not show that one on the home screen do we? I don't think that's an option that needs to be in the front seat.

**Comment by ksooo** [comment]
@tamland ping. just an observation, no idea how a proper fix must look like: https://github.com/xbmc/xbmc/blob/master/xbmc/filesystem/AddonsDirectory.cpp#L472 creating a non-folder items leads to "PlayMedia" builtin called for this item.

**Comment by tamland** [comment]
What 'tile' are we talking about..?

**Comment by Razzeee** [comment]
New home screen in the second branch of this repo

**Comment by tamland** [comment]
Yeah.. I've said it before, you can't just take a random internal path, put it in directory provider and expect it work. You have to add support for it. Most likely the zip install is handled by the addon browser window.

**Comment by phil65** [comment]
@tamland Would it be possible to add support for it or should i switch back to static items instead of VFS?

**Comment by tamland** [comment]
Sure, it's possible to make it work with directory providers. Moving the things from https://github.com/xbmc/xbmc/blob/master/xbmc/addons/GUIWindowAddonBrowser.cpp to CAddonsDirectory is probably enough. And you have think about how to handle change detection ofc. and if the current one is sufficient..

**Comment by phil65** [comment]
Will you have time to have a go at it? Otherwise I will probably go for the quick solution for now.
Thx in advance.

**Comment by tamland** [comment]
I'll file it in my todo list together with adding support for the repository directories which @HitcherUK added earlier;)

**Comment by phil65** [comment]
cool, thx a lot!

**Comment by Razzeee** [comment]
I still think it shouldn't be on the homescreen at all :X

**Comment by ksooo** [comment]
> I still think it shouldn't be on the homescreen at all

 why? i think it should. I'm using it very often to install my own (not yet released or heavily patched) shit.

**Comment by Razzeee** [comment]
I do too, but we're devs and we're not building a media center for devs, are we? I don't think a feature we just added a warning sign before should be front and center on the home screen.

**Comment by ksooo** [comment]
> we're not building a media center for devs

we do! ;-)

**Comment by da-anda** [comment]
yes, should IMO be removed from home screen. I also use it, but it's more a "pro" feature and should be burried somewhere, like before.

**Comment by ksooo** [comment]
> yes, should IMO be removed from home screen. I also use it, but it's more a "pro" feature and should be burried somewhere, like before.

 imo not.

**Comment by Mike-KL** [comment]
After trying v2 and reporting in following thread, to be told known issue and to look here; I fully agree with Razzeee comments above that option should not be on home screen at top level.

http://forum.kodi.tv/showthread.php?tid=262144&pid=2463075#pid2463075

**Comment by ksooo** [comment]
@tamland any chance for a fix for this for Krypton (regardless whether it is removal or keep of the item in question)?

**Comment by tamland** [comment]
Don't know when I'll have time yet (and if we're removing zip it's more work finding a new place for it). But didn't we agree not to use this vfs path before it's been actually implemented?

**Comment by phil65** [comment]
@tamland just give me a heads-up whether it will get done in time for krypton, otherwise i will change that list to static items. Thx!
